hub/saml-engine/src/test/java/uk/gov/ida/hub/samlengine/validation/country/EidasAttributeStatementAssertionValidatorTest.java

```diff
@@ -88,6 +90,26 @@ public void shouldThrowIfAttributeValueHasInvalidSchemaType() {
         validator.validate(assertion);
     }
 
+    @Test(expected = SamlTransformationErrorException.class)
+    public void shouldThrowIfAttributeValueHasNullSchemaType() {
+        XMLObject xmlObject = mock(XMLObject.class);
+        when(firstName.getAttributeValues()).thenReturn(List.of(xmlObject));
+        when(xmlObject.getSchemaType()).thenReturn(null);
+
+        validator.validate(assertion);
+    }
+
+    @Test(expected = SamlTransformationErrorException.class)
+    public void shouldNotThrowIfAttributeValueHasNullSchemaNamespace() {
```

Comment:
`@Test(expected = ...)` indicates the test should throw an exception, but the name suggests it shouldn't (`shouldNotThrow...`). Which is it?